### DISCUSSION TITLE
The service refused the connection - perhaps it is offline (N8N and NocoDB)

### DISCUSSION THREAD

**Discussion by iphillz** [nocodb/nocodb#7115] [Q&A] [answered]
![image](https://github.com/nocodb/nocodb/assets/152276393/ae6aeddf-bca8-468a-b914-06711e5bf28a)

I have just freshly installed Docker, where I run N8N and NocoDB. 

The issue is that I need help connecting both together. Getting error _**The service refused the connection - perhaps it is offline**_

The same issue was if I tried connecting N8N with supabase or Postgress. What is the deal with N8N and NocoDB. ?

Maybe some firewall? I did open 5678 and 8080 port. 

P.S. Tried to connect NocoDB via API token and also via Auth token, both times same error.

**Comment by mertmit** [comment, marked as answer]
You need to use host ip address or alias in order to access host services from container ([reference](https://docs.docker.com/desktop/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host)).
So can you try using host.docker.internal instead of localhost.

**Comment by iphillz** [comment]
Worked. Thank you @mertmit 

![image](https://github.com/nocodb/nocodb/assets/152276393/9d441983-1337-42b3-9c04-952d3ffabe38)
